src/pages/ClinicalHistory.jsx

```diff
@@ -0,0 +1,196 @@
+import { useState } from 'react';
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card';
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '@/components/ui/table';
+import { Badge } from '@/components/ui/badge';
+import { FileIcon, ImageIcon, DownloadIcon } from 'lucide-react';
+import { Button } from '@/components/ui/button';
+
+export function ClinicalHistory() {
+  const [conditions] = useState([
+    {
+      name: 'Hypertension',
+      details: 'High blood pressure',
+      since: '01-01-2020'
+    },
+    {
+      name: 'Diabetes',
+      details: 'High blood sugar',
+      since: '01-01-2020'
+    }
+  ]);
+
+  const [treatments] = useState([
+    {
+      name: 'Insulin',
+      startDate: '01-01-2020',
+      endDate: '01-01-2021',
+      instructions: 'Inject once a day'
+    },
+    {
+      name: 'Metformin',
+      startDate: '01-01-2020',
+      endDate: '01-01-2021',
+      instructions: 'Take with food'
+    }
+  ]);
+
+  const [analytics] = useState([
+    {
+      name: 'analysis-172398379487.pdf',
+      originalName: 'analysis.pdf',
+      url: 'https://example.com',
+      date: '01-01-2023'
+    },
+    {
+      name: 'analysis2-172398379487.pdf',
+      originalName: 'analysis2.pdf',
+      url: 'https://example.com',
+      date: '01-02-2023'
+    },
+  ]);
+  
+  const [images] = useState([
+    {
+      name: 'resonance-172398379487.fh7',
+      originalName: 'resonance.fh7',
+      url: 'https://example.com',
+      date: '01-01-2023'
+    },
+    {
+      name: 'xray-172398379487.png',
+      originalName: 'xray.png',
+      url: 'https://example.com',
+      date: '01-02-2023'
+    },
+  ]);
+
+  const [allergies] = useState(['Peanuts', 'Penicillin']);
```

Comment:
I do not see a "setX". If these are to be static, consider using a constant insted of an state for performance reasons